src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykTreeCellRenderer.kt

```diff
@@ -111,24 +112,13 @@ class SnykTreeCellRenderer : ColoredTreeCellRenderer() {
                 val issue = value.userObject as ScanIssue
                 nodeIcon = SnykIcons.getSeverityIcon(issue.getSeverityAsEnum())
                 val range = issue.range
-                var showIgnoredLabel = false
```

Comment:
Moved most of this logic to the issue definition, so we can contain the product-based switching into one place and not clutter this code